lib/raxx/http1.ex

```diff
@@ -723,7 +723,7 @@ defmodule Raxx.HTTP1 do
     payload_headers =
       case content_length(headers) do
         nil ->
-          # NOTE `:erlang.iolist_size/1` acceps binaries, i.e. should be `:erlang.iodata_size/1`
+          # NOTE `:erlang.iolist_size/1` accepts binaries, i.e. should be `:erlang.iodata_size/1`
```

Comment:
Turns out this was just an incorrect comment ` :erlang.iolist_size("")` works fine.
Thanks for bringing it to my attention. will close this PR and delete the comment.